src/server/v2.0/handler/auditlog.go

```diff
@@ -21,10 +21,9 @@ type auditlogAPI struct {
 }
 
 func (a *auditlogAPI) ListAuditLogs(ctx context.Context, params auditlog.ListAuditLogsParams) middleware.Responder {
-	// ToDo enable permission check
-	// if !a.HasPermission(ctx, rbac.ActionList, rbac.ResourceLog) {
-	//	return a.SendError(ctx, ierror.ForbiddenError(nil))
-	// }
+	if err := a.RequireSysAdmin(ctx); err != nil {
```

Comment:
Non-admin users also need to list the related logs

Reply:
why no-admin users can see the global logs? He can know the operation of other projects?
It's not make sense

Reply:
Login as non-admin users, there is an item to show all logs related with the user on the left panel on portal

Reply:
ok,I got it. So only verify the authN?

Reply:
Please refer to the implement of 1.10